xray/commands/curation/audit.go

```diff
@@ -279,24 +277,34 @@ func printResult(format utils.OutputFormat, projectPath string, packagesStatus [
 
 func convertToPackageStatusTable(packagesStatus []*PackageStatus) []PackageStatusTable {
 	var pkgStatusTable []PackageStatusTable
-	for _, pkgStatus := range packagesStatus {
+	for index, pkgStatus := range packagesStatus {
+		// We use auto-merge supported by "go-pretty" library. It doesn't have an option to merge lines by group of uniq fields.
+		// we make each group to be merged only with itself by adding/not adding space, this way it won't be merged with the next group.
```

Comment:
Since thw above comment isn't perfect in terms of English grammar, let's run it using a brammar corerction tool.
For example - 
... supported by "go-pretty" library... --> ... supported by the "go-pretty" library...